README.md

```diff
@@ -245,7 +245,7 @@ func main() {
 
 	// Create a Filter struct with criteria you would like to filter
 	// The full struct definition can be found here for all of the supported filters:
-	// https://github.com/aws/amazon-ec2-instance-selector/blob/master/pkg/selector/types.go
+	// https://github.com/aws/amazon-ec2-instance-selector/blob/main/pkg/selector/types.go
```

Comment:
master's also used in the Contributing section at the bottom of the readme

Reply:
goood catch-- would you mind merging this one first then either you or I can re-open another PR to address those? I made this change via GitHub cuz I thought it wouldn't need any updates :(